### DISCUSSION TITLE
Netbox Proxbox Plugin

### DISCUSSION THREAD

**Discussion by TheDmDLP** [netbox-community/netbox#14748] [Help Wanted!]
I am trying to set up the Proxbox plugin and always fail at the connection between netbox and proxmox. 
I am running a 4 node cluster and would like to have all the info of the VM's and containers documented in Netbox. I have entered a node in the config. It is not quite clear to me whether I have to enter all 4 nodes in the config or whether one is enough. I have created an API key and given it the right to read in proxmox. 
However, when I click on Proxmox full update under plugins, the error message appears:
**The request failed with code 403 Forbidden: {'detail': 'You do not have permission to perform this action.'}.**
I don't want to give the Netbox user write permissions in Proxmox for security reasons. 
Thx 4 your Help

**Comment by candlerb** [comment]
I'd say you'd best ask this on the issue tracker or discussions for the Proxbox project, because it's not something that most Netbox users would use.

**Reply by TheDmDLP** [reply]
i try and update this thread if i got answerd
